Modules/main.c

```diff
@@ -621,6 +623,7 @@ pymain_free(void)
     _PyPathConfig_ClearGlobal();
     _Py_ClearStandardStreamEncoding();
     _Py_ClearArgcArgv();
+    _Py_Deepfreeze_Fini();
```

Comment:
Is this where @vstinner said it should be called?

Reply:
It is still to be decided but it worked here on Linux so I put it here, can be changed easily.